Added note to tutorial on Capitalized component variable names. (#4505)
sveltejs/svelte: site/content/tutorial/01-introduction/05-nested-components/text.md

<Nested/>
```

Notice that even though `Nested.svelte` has a `<p>` element, the styles from `App.svelte` don't leak in.

> **NOTE**: The component variable names (such as `Nested`) should be Capitalized

Reviewer: ```suggestion
Notice that the component name `Nested` is capitalized. This format has been adopted to provide a differentiation between user-defined components and regular HTML tags.
```

Author: Thank you for your review. I have added your suggestions.

<Nested/>
```

Notice that even though `Nested.svelte` has a `<p>` element, the styles from `App.svelte` don't leak in.

> Notice that the component name `Nested` is capitalized. This format has been adopted to provide a differentiation between user-defined components and regular HTML tags.

Reviewer: ```suggestion
Also notice that the component name `Nested` is capitalized. This convention has been adopted to allow us to differentiate between user-defined components and regular HTML tags.
```
